src/JoomlaBrowser.php

```diff
@@ -804,7 +804,7 @@ public function checkExistenceOf($name)
 	public function checkAllResults()
 	{
 		$this->debug("Selecting Checkall button");
-		$this->click(['xpath' => "//thead//input[@name='checkall-toggle' or @name='toggle']"]);
```

Comment:
This change should actually not cause a failure, since we have an "OR" condition